lib/components/form/batch-settings-panel.js

```diff
@@ -0,0 +1,111 @@
+// import {DropdownSelector} from '@opentripplanner/trip-form'
```

Comment:
Remove commented import.

Reply:
We may end up using these selectors instead, so I'd rather leave the reference instead of having to dig up where that component lives (it took a few minutes last time).